test/cli/package-prefix-enforcement/test.out

```diff
@@ -8,20 +8,32 @@ nested/nested.rb:5: File belongs to package `Root::Nested` but defines a constan
 nested/nested.rb:40: Class or method behavior may not be defined outside of the enclosing package namespace `Root::Nested` https://srb.help/3713
     40 |  sig {returns(NilClass)}
           ^^^^^^^^^^^^^^^^^^^^^^^
-  Note:
-    Attempting to define class or method behavior in package namespace `Root`
+    nested/nested.rb:36: Attempting to define class or method behavior in `Root`
+    36 |module Root
+               ^^^^
+    __package.rb:3: Enclosing package declared here
+     3 |class Root < PackageSpec
+        ^^^^^^^^^^^^^^^^^^^^^^^^
 
 nested/nested.rb:41: Class or method behavior may not be defined outside of the enclosing package namespace `Root::Nested` https://srb.help/3713
     41 |  def self.method
           ^^^^^^^^^^^^^^^
-  Note:
-    Attempting to define class or method behavior in package namespace `Root`
+    nested/nested.rb:36: Attempting to define class or method behavior in `Root`
+    36 |module Root
+               ^^^^
+    __package.rb:3: Enclosing package declared here
+     3 |class Root < PackageSpec
+        ^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
It looks like we're using "enclosing" in two different ways here:

- the error header says that the "enclosing package namespace" is `Root::Nested`
- this error line says that the "enclosing package" is `Root`